docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -236,20 +322,20 @@ need to view monitoring data for APM Server:
 |`monitoring_user`
 |Grants access to monitoring indices for APM Server
 |====
+. Assign <<apm-feature-roles,additional APM feature roles>> to users as needed
+(including the _Central configuration management role_, which is <<apm-central-config-role-note,required in most cases>>).
 
 ////
 ***********************************  ***********************************
 ***********************************  ***********************************
 ////
 
+[float]
 [[apm-privileges-api-key]]
-=== Grant privileges and roles needed for API key management
-
-++++
-<titleabbrev>Create an _API key_ user</titleabbrev>
-++++
+=== Create an _API key_ role
```

Comment:
@endorama to confirm if this should be mark as deprecated

Reply:
Yes, it should be. Full context in  https://github.com/elastic/observability-docs/pull/4193#issuecomment-2363155509